posthog/api/notebook.py

```diff
@@ -171,6 +232,26 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                 queryset = queryset.filter(last_modified_at__gt=relative_date_parse(request.GET["date_from"]))
             elif key == "date_to":
                 queryset = queryset.filter(last_modified_at__lt=relative_date_parse(request.GET["date_to"]))
+            elif key == "contains":
+                contains = request.GET["contains"]
+                match_pairs = contains.replace(",", " ").split(" ")
+                # content is a JSONB field that has an array of objects under the key "content"
+                # each of those (should) have a "type" field
+                # and for recordings that type is "ph-recording"
+                # each of those objects can have attrs which is a dict with id for the recording
+                for match_pair in match_pairs:
+                    target, match = match_pair.split(":")
+                    if target == "recording":
```

Comment:
Feels like we walked up to the finish line and then stopped 😅  - why don't we allow targets other than recording? The rest of the code basically should already work for featureflags, Persons, insights etc...

Reply:
Yep absolutely.... For a follow-up IMHO